CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+Developing DVOL
+---------------
+
+If you wish to work on DVOL, you'll first need [Go](http://www.golang.org) installed on your machine (version 1.6+ is *required*).
+
+For local dev first make sure Go is properly installed, including setting up a [GOPATH](http://golang.org/doc/code.html#GOPATH). You will also need to add `$GOPATH/bin` to your `$PATH`.
+
+
+Next, using [Git](https://git-scm.com/), clone this repository into `$GOPATH/src/github.com/ClusterHQ/dvol`. All the necessary Golang dependencies are vendored, however you will need to install some Python dependancies and Golang tools -
+
+```sh
+$ make bootstrap
```

Comment:
:question: potentially add a `cd $GOPATH/src/github.com/ClusterHQ/dvol` to the start of this.